pkg/config/configuration.go

```diff
@@ -333,11 +336,17 @@ type ObservabilityParameters struct {
 	//default is false. if false, enable trace at booting
 	DisableTrace bool `toml:"disableTrace"`
 
-	//default is InternalExecutor. if InternalExecutor, use internal sql executor, FileService will implement soon.
+	//default is FileService. if InternalExecutor, use internal sql executor, FileService will implement soon.
```

Comment:
Go comments should like "BatchProcessor default is FileService. if InternalExecutor, use internal sql executor, FileService will implement soon". Begin with the name of the exported variable or function